### PR TITLE
IBA-61: Added region to user & tenant mode and requests

### PR DESCRIPTION
**Pull request by onlythenaive**

### CODE REVIEW

File: hub/hub-api.yaml [hunk 1]
```diff
@@ -1767,6 +1773,8 @@ components:
           description: The list of subscriptions enabled for the user.
           items:
             $ref: '#/components/schemas/Subscription'
+        region:
+          type: string
```
Comment (aehrenthal) [inline hub/hub-api.yaml:1777]: Shouldnt it be an Enum here too?

File: hub/hub-api.yaml [hunk 2]
```diff
@@ -1933,6 +1941,8 @@ components:
           description: The list of features available to the tenant.
           items:
             type: string
+        region:
+          type: string
```
Comment (aehrenthal) [inline hub/hub-api.yaml:1945]: Shouldnt it be an Enum here too?

**Review by aehrenthal — CHANGES_REQUESTED** [review]

**Review by olyavertwist — CHANGES_REQUESTED** [review]
Ilia, could you please add this description to all the instances of the region parameter (including User, UserRequest, UserAccountCreationRequest, etc. objects):

description: The target region of the user account. Inperium stores data in the datacenters on the account location in order to ensure the fastest access to data and its security.

File: hub/hub-api.yaml (hunk 1 above)
Comment (BullDog57Rus) [reply hub/hub-api.yaml:1777]: we'd better to move region to the whole component like `Region` and reuse it everywhere

**Pull request closed without merging**